sdk/loadtesting/azure-developer-loadtesting/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Add NOT_VALIDATED to the list of terminal states for the file validation poller.
```

Comment:
I think this should go into either `Breaking Changes` or `Bugs Fixed` whether this was a bug fix for something the service always supported or a breaking change in behavior.

Reply:
This would classify neither as a bug, nor as a breaking change.
Currently, the `if condition` of the changes made would never be satisfied